src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -117,6 +117,149 @@ public class XpathRegressionTest extends AbstractModuleTestSupport {
 
     private static final Set<String> INTERNAL_MODULES = getInternalModules();
 
+    // Checks whose files need to be renamed to new pattern "InputXpath{Check}Xxx.java"
+    // till checkstyle issue
+    private static final Set<String> RENAME_INPUT_XPATH = Set.of(
```

Comment:
> We are then good to completely swap old enforcement with new as it will be fixed as your list is worked on. The list must reference the issue to work on (until https://....) and complete it. I recommend to make this a new issue, we can mark it as easy

After your confirmation, I will create a new issue as suggested at https://github.com/checkstyle/checkstyle/pull/14595#discussion_r1518454112 and update comment above